theano/gpuarray/tests/test_linalg.py

```diff
@@ -191,3 +199,100 @@ def test_invalid_input_fail_negative_definite(self):
         A_val = -M_val.dot(M_val.T)
         fn = self.get_gpu_cholesky_func(True, False)
         self.assertRaises(LinAlgError, fn, A_val)
+
+
+class TestMagma(unittest.TestCase):
+
+    def setUp(self):
+        if not config.magma.enabled:
+            self.skipTest('Magma is not enabled, skipping test')
+
+    def test_gpu_matrix_inverse(self):
+        A = theano.tensor.fmatrix("A")
+
+        fn = theano.function([A], gpu_matrix_inverse(A), mode=mode_with_gpu.including('magma'))
```

Comment:
I removed that. My original idea was that the user can enable magma by enabling its optimizer. But approach with config `magma.enable` seems better.